applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/prj_fast_pair.conf

```diff
@@ -0,0 +1,145 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# ZDebug: Fast Pair version
+#
+# This ZDebug build type configuration enables Fast Pair support.
```

Comment:
I assume it is a convention to call it `ZDebug` only here while we call those configs just `debug` everywhere else?

Reply:
I will align it too (you are right that it may be misleading)